doc/internal-api/doxygen.cfg.in

```diff
@@ -39,17 +39,21 @@ INPUT                  = \
   src/libcmd \
   src/libexpr \
   src/libexpr/flake \
-  src/libexpr/tests \
-  src/libexpr/tests/value \
+  src/libexpr-tests \
+  src/libexpr-tests/value \
+  src/libexpr-tests/test \
+  src/libexpr-tests/test/value \
```

Comment:
Discussed in the team meeting:
```suggestion
  tests/libexpr/tests/value \
```

While the nested `tests` looks silly, it makes the diff within the files smaller.